### ISSUE TITLE
LTR area within RTL note

### ISSUE THREAD

**Issue by Yakikim** [esm7/obsidian-rtl#21]
Hey, 
First of all, thanks a lot!
second, I did some workaround of this issue, using the plug in called Admonition. 
I can write some guide if you want. 
The final result of this workaround is:
![image](https://user-images.githubusercontent.com/60339558/131666080-2d0a3963-2cff-4192-8897-b2fb1b5e54cf.png)

BR, Yaki

**Comment by esm7** [comment]
First, what a beautiful usage of RTL in Obsidian! It's such a pleasure to see.
Second, it's surely a very useful tip for those who target Preview mode. If you want to write a concrete example with a screenshot that I can put in the Readme, I'm sure many users will find it useful.

**Comment by omergunr100** [comment]
Thank you both for an incredible plugin, and a great guide!

Truly a lifesaver, I was messing with a .svg with LTR elements in an RTL page for about 2 hours until I found this and solved my issue immediately :)

Before:
![image](https://user-images.githubusercontent.com/24607757/166114027-44f1ac97-be1e-4499-b1e0-70117b4c481f.png)


After:
![image](https://user-images.githubusercontent.com/24607757/166114005-fb62ecf6-8f87-4fe9-8fd1-13d1782b89c0.png)

**Comment by Yakikim** [comment]
Thanks! I'm happy you've found it helpful. I've added it also to my blog here: https://yakis.netlify.app/ltr%20area%20within%20rtl%20note/

**Comment by chromer030** [comment]
Is this mixed RTL and LTR usage ?

https://forum.obsidian.md/t/mixed-rtl-and-ltr/34808

**Comment by Yakikim** [comment]
Indeed. Review the workaround here:https://yakis.netlify.app/ltr%20area%20within%20rtl%20note/
